ui/component/copyableStreamkeyUnmask/view.jsx

```diff
@@ -0,0 +1,41 @@
+// @flow
+import Button from 'component/button';
+import React from 'react';
+
+type Props = {
+  snackMessage: ?string,
+  doToast: ({ message: string }) => void,
+  label?: string,
+  primaryButton?: boolean,
+  name?: string,
+};
+
+export default function CopyableText(props: Props) {
+  const { doToast, primaryButton = false } = props;
+
+  function showStreamkey() {
+    if (document.getElementById('livestream-key').type === 'password') {
+      doToast({ message: __('Showing Stream Key') });
+      document.getElementById('livestream-key').setAttribute('type', 'text');
+      document.getElementById('streamkey-unmask').style.color = 'white';
+      document.getElementById('streamkey-unmask').innerText = 'Hide Stream key';
+    } else if (document.getElementById('livestream-key').type === 'text') {
+      doToast({ message: __('Hiding Stream Key') });
+      document.getElementById('livestream-key').setAttribute('type', 'password');
+      document.getElementById('streamkey-unmask').style.color = 'white';
+      document.getElementById('streamkey-unmask').innerText = 'Show Stream key';
+    }
+  }
```

Comment:
It's unnecessary to create a separate Component just for unmask functionality, and it is also not the React-way to tweak the DOM directly for something that React supports through States.

- The button and the unmasking button can be encapsulated within the new `copyableStreamkey` itself. 
- Optionally, for an even cleaner solution, we don't have to create `copyableStreamkey`, but augment the original `copyableText` to support an optional parameter `isPassword` (or `enableMask` or something).

Reply:
I removed the Component entirely.